HW_Sem1/Task1.py

```diff
@@ -0,0 +1,22 @@
+# Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей. 
+# Дано a, b, c - стороны предполагаемого треугольника. Требуется сравнить длину каждого отрезка-стороны 
+# с суммой двух других. Если хотя бы в одном случае отрезок окажется больше суммы двух других, 
+# то треугольника с такими сторонами не существует. Отдельно сообщить является ли треугольник разносторонним, 
+# равнобедренным или равносторонним.
+
+A_SIDE = 10
+B_SIDE = 10
+C_SIDE = 10
+
+result = None
+
+if A_SIDE > (B_SIDE + C_SIDE) or B_SIDE > (A_SIDE + C_SIDE) or C_SIDE > (A_SIDE + B_SIDE):
+    result = 'Такого треугольника не существует'
+else:
+    if A_SIDE == B_SIDE and B_SIDE == C_SIDE:
+        result = 'Это равносторонний треугольник'
+    elif A_SIDE == B_SIDE or B_SIDE == C_SIDE or C_SIDE == A_SIDE:
+        result = 'Это равнобедренный треугольник'
+    else:
+        result = 'Это разносторонний треугольник'
```

Comment:
Тут тоже можно иначе сделать, чтобы уменьшить код.
```suggestion
    elif A_SIDE != B_SIDE != C_SIDE:
        result = 'Это разносторонний треугольник'
    else:
        result = 'Это равнобедренный треугольник'
```

Reply:
С равнобедренным не работает(

Reply:
elif A_SIDE != B_SIDE != C_SIDE != A_SIDE: